### ISSUE TITLE
How do I get the segmentation annotations only for the 50k validation split of the original ImageNet?

### ISSUE THREAD

**Issue by NightMachinery** [LUSSeg/ImageNet-S#8]
How do I get the segmentation annotations only for the 50k validation split of the original ImageNet?

I only need a few thousand images, so I don't want to download the full ImageNet dataset (~150GB). I am downloading only its 50k validation split instead:

https://academictorrents.com/details/5d6d0df7ed81efd49ca99ea4737e0ae5e3a5f2e5

How do I get the segmentation annotations for this split?

**Comment by gasvn** [comment]
You can remove the training dataset in https://github.com/LUSSeg/ImageNet-S/blob/main/datapreparation/data_preparation.sh The val in imagenet-s is a part of imagenet dataset, and we keep the result of the val in imagenet as the test set in imagenet-s.

**Comment by NightMachinery** [comment]
Thanks, I loaded the data successfully.
